File: DuzensizArttirma/DuzensizArttirma.java
package duzensizarttirma;

public class DuzensizArttirma {

    public static void main(String[] args) {
        int sonuc = 0, artis = 0;
        boolean yon = false;
        for (int i = 1; i <= 35; i += artis) {
            sonuc += i;
            if (i != 35) {
                System.out.print(i + " + ");
            } else {
                System.out.print(i + " = " + sonuc + "\n");
            }
            if (yon) {
                if (artis > 2) {
                    artis--;
                } else {
                    yon = false;
                    artis--;
                }
            } else {
                if (artis < 4) {
                    artis++;
                } else {
                    yon = true;
                    artis++;
                }
            }
        }
    }
}

File: DuzensizArttirma/DuzensizArttirma.py
def duzensiz_arttirma():

    yon = True
    toplam = 0
    sayi = 1
    artirma = 0

    for i in range(13):

        sayi += artirma

        if sayi != 35:
            print(sayi, end=" + ")
        else:
            print(sayi, end=" = ")

        if yon == True:
            if artirma < 5:
                artirma += 1
            else:
                yon = False
                artirma -= 1
        else:
            if artirma > 1:
                artirma -= 1
            else:
                yon = True
                artirma += 1
        toplam += sayi

    return toplam

print(duzensiz_arttirma())